src/VegaLite.jl

```diff
@@ -27,11 +27,11 @@ export @vg_str, @vgplot, vgplot, @vgfrag, vgfrag
 export load, save
 export deletedata, deletedata!
 
-const vegaliate_app_path = artifact"vegalite_app"
-const vegaliate_app_includes_canvas = ispath(joinpath(vegaliate_app_path, "node_modules", "canvas"))
+vegaliate_app_path(args...) = joinpath(artifact"vegalite_app", args...)
+const vegaliate_app_includes_canvas = ispath(vegaliate_app_path("node_modules", "canvas"))
```

Comment:
I think this might not yet be portable? I think what is happening here is that it will "bake" the status of that into the compiled code as well, which is probably not what we want.

I think what we want to do here is change this to
```julia
const vegaliate_app_includes_canvas = Ref{Bool}()
```
and then add a `__init__` function:
```julia
function __init__()
    vegaliate_app_includes_canvas[] = ispath(vegaliate_app_path("node_modules", "canvas"))
end
````
And then you'll need to change all instances where `vegaliate_app_includes_canvas` is accessed into `vegaliate_app_includes_canvas[]`.